torchtitan/experiments/torchcomms/README.md

```diff
@@ -1,25 +1,32 @@
 ## TorchTitan & TorchComms Composability Testing
 
-#### Overview
+### Overview
 
-This folder provides a framework for composability testing with TorchComms and distributed training in TorchTitan. The goal is to enable flexible experimentation with distributed communication primitives and parallelism strategies in PyTorch.
-TODO: add more explanation once the torchcomm goes public.
----
-#### Example
+This folder provides a framework for composability testing with TorchComms and distributed training in TorchTitan. It enables flexible experimentation with distributed communication primitives and various parallelism strategies in PyTorch.
+
+> **TODO:** Additional documentation will be provided once TorchComms is publicly released.
+
+### Quick Start
+
+The following command uses Llama 3 as an example, but should work with all models:
```

Comment:
Are we sure it work with all models?? For example we have not enable EP yet, so deepseek is not working?